Dockerfile.slim

```diff
@@ -3,11 +3,12 @@ LABEL maintainer="sig-platform@spinnaker.io"
 
 ENV KUSTOMIZE_VERSION=3.8.6
 ENV KUSTOMIZE4_VERSION=4.5.5
-ENV PACKER_VERSION=1.8.1
+ENV PACKER_VERSION=1.10.1
 ENV HELMFILE_VERSION=0.153.1
 
 
 ARG TARGETARCH
+ARG PLUGINS="amazon azure docker googlecompute"
```

Comment:
I'm not TERRIBLY comfortable with the docker plugin - as that TYPICALLY requires host level access kinda thing in order to access the docker socket/daemon/etc.  kinda stuff and exposes things like that runc vulnerability as I recall.  

```The Docker builder must run on a machine that has Docker Engine installed. Therefore the builder only works on machines that support Docker and does not support running on a Docker remote host. You can learn about what [platforms Docker supports and how to install onto them](https://docs.docker.com/engine/installation/) in the Docker documentation.``` FYI on the docker builder.

Reply:
I'm fine with just removing it

Reply:
Removed (and I've updated the PR description)